CMakeLists.txt

```diff
@@ -320,7 +325,10 @@ endif()
 # On Android RELEASE builds, we disable exceptions and RTTI to save some space (about 75 KiB
 # saved by -fno-exception and 10 KiB saved by -fno-rtti).
 if (ANDROID OR IOS OR WEBGL)
-    set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -fno-exceptions -fno-unwind-tables -fno-asynchronous-unwind-tables -fno-rtti")
```

Comment:
Wdyt of simply removing `IOS` from here? We won't need to toggle and create different builds.

Reply:
I want to keep it an upstreamable PR, that is why I introduced a new flag used only by us.

Reply:
The flag `FILAMENT_ENABLE_CPP_EXCEPTIONS` is quite confusing in this form: 
- It affects debug and release builds of Windows if you use GCC, but ignored in case of MSVC,
- It affects release builds of Android, iOS+Catalyst, and WebGL, but ignored in debug,
- It's ignored on macOS.

So I think if we introduce such a high level flag, it should be handled independently of the current defaults.